src/rpc.rs

```diff
@@ -176,28 +172,32 @@ pub enum RpcClient {
 
 impl RpcClient {
     /// Start a JSON-RPC request.
-    pub async fn request<T: DeserializeOwned>(
+    pub async fn request<'a, T: DeserializeOwned + std::fmt::Debug>(
```

Comment:
added debug bound here to make is possible to log here but might be more readable logging on the the concrete type